string name
Vector2f center
float32 radius
float32 a1
float32 a2
float32 thickness
